collect_app/src/main/java/org/odk/collect/android/fragments/dialogs/FormsDownloadResultDialog.kt

```diff
@@ -7,12 +7,11 @@ import android.os.Bundle
 import androidx.fragment.app.DialogFragment
 import com.google.android.material.dialog.MaterialAlertDialogBuilder
 import org.odk.collect.android.R
-import org.odk.collect.android.errors.ErrorActivity
-import org.odk.collect.android.errors.ErrorItem
+import org.odk.collect.errors.ErrorActivity
 import java.util.ArrayList
 
 class FormsDownloadResultDialog : DialogFragment() {
-    private lateinit var failures: ArrayList<ErrorItem>
+    private lateinit var failures: ArrayList<org.odk.collect.errors.ErrorItem>
```

Comment:
You can add import for the full path and here use only `ErrorItem`. The same in many other places.

Reply:
Oops! This looks like something that got added while moving the class.